python/hsfs/feature_view.py

```diff
@@ -161,7 +154,14 @@ def get_batch_query(
             `str`: batch query
         """
         return self._feature_view_engine.get_batch_query_string(
-            self, start_time, end_time
+            self,
```

Comment:
New parameter still needs to be added to the docstring,

Reply:
Do you mean td version? It is not required in the parameter.

Reply:
I was anticipating that you are going to add the td version parameter to that function, based on Jim's comment and the fact that the Java API getBatchQuery method has that version parameter. I don't know what the conclusion was between you and Jim was, but I think the API should be consistent between Java and Python.